auslib/log.py

```diff
@@ -138,7 +138,7 @@ def safer_format_traceback(exc_typ, exc_val, exc_tb):
 def configure_logging(stream=sys.stdout, formatter=JsonLogFormatter, format_=log_format, level=logging.DEBUG):
     logging.setLoggerClass(BalrogLogger)
     handler = logging.StreamHandler(stream)
-    formatter = formatter(fmt=format_)
+    formatter = JsonLogFormatter(fmt=format_)
```

Comment:
This looks it will regress the LOG_FORMAT=plain setting docker uses. Otherwise fine.